jobs/gradle.yml

```diff
@@ -0,0 +1,50 @@
+# gradle
+#
+# Job responsible for building Gradle Wrapper based projects.
+#
+
+parameters:
+  jdkVersions:
+    - name: JDK_8
+      version: "1.8"
+    - name: JDK_11
+      version: "1.11"
```

Comment:
version is not 1.11. Oracle changed versioning with Java 9. It's just 11 now.

Reply:
I knew that you will comment on this. It is unfortunately `1.11` in the implementation of the `GradleTaskV2`. I checked their code:

https://github.com/microsoft/azure-pipelines-tasks/pull/8716/files#diff-ada8bc0a0c39c3b4dab9db5e8f051562R200

Reply:
I see. Thanks! It's very strange that very old naming was used here and no prover configuration is provided still.

Reply:
Yes, no idea why they choose that. Believe me, I tried "1.8" and "11" and wondered why the latter was not working. So started digging in the code and found this :)